### PR TITLE
Root

### PR DESCRIPTION
**Pull request by ms202202**

### CODE REVIEW

**Review by Stericson — APPROVED** [review]

**Review by Aidenmichael — COMMENTED** [review]
Root please 

`

**Review by Aidenmichael — COMMENTED** [review]
How does this work?

File: README.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# RootTools
+RootTools Library
```
Comment (Danielversailles97) [inline README.md:2]: 6dc1b979932858c189f864f6d03b7ef6498e3748

File: README.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# RootTools
```
Comment (Danielversailles97) [inline README.md:1]:
```

```

**Review by Danielversailles97 — CHANGES_REQUESTED** [review]

**Review by Danielversailles97 — APPROVED** [review]

**Pull request merged**